spec/jobs/document_proofing_job_spec.rb

```diff
@@ -24,6 +24,7 @@
       ssn: '123456789',
       phone: '18888675309',
       state: 'MT',
+      state_id_type: 'drivers_license',
```

Comment:
There's a couple assertions in this file which I think we can update to check that the ID type is actually being logged:

https://github.com/18F/identity-idp/blob/e5b72af77401c00508cefabad56072ca054923e8/spec/jobs/document_proofing_job_spec.rb#L147

https://github.com/18F/identity-idp/blob/e5b72af77401c00508cefabad56072ca054923e8/spec/jobs/document_proofing_job_spec.rb#L197

`have_logged_event` only checks a partial match, which is why it passes as-is.